Userland/Utilities/sort.cpp

```diff
@@ -106,25 +107,27 @@ ErrorOr<int> serenity_main([[maybe_unused]] Main::Arguments arguments)
     args_parser.add_option(options.numeric, "treat the key field as a number", "numeric", 'n');
     args_parser.add_option(options.separator, "The separator to split fields by", "sep", 't', "char");
     args_parser.add_option(options.reverse, "Sort in reverse order", "reverse", 'r');
+    args_parser.add_option(options.zero_terminated, "Use '\\0' as the line delimiter instead of a newline", "zero-terminated", 'z');
     args_parser.add_positional_argument(options.files, "Files to sort", "file", Core::ArgsParser::Required::No);
     args_parser.parse(arguments);
 
+    StringView line_delimiter { options.zero_terminated ? "\0" : "\n", 1 };
```

Comment:
Can this just be:
```c++
auto line_delimiter = options.zero_terminated ? "\0"sv : "\n"sv;
```

I think our `sv` is able to count the null byte in the length, i.e. this builds for me:
```c++
constexpr auto s = "\0"sv;
static_assert(s.length() == 1);
```

Reply:
That's better, thanks :^)